app/build.gradle

```diff
@@ -23,15 +23,15 @@ buildscript {
 }
 
 android {
-    compileSdkVersion Versions.compile_sdk
-    buildToolsVersion Versions.build_tools
+    compileSdkVersion build_versions.compile_sdk
+    buildToolsVersion build_versions.build_tools
     defaultConfig {
         applicationId "org.mozilla"
-        minSdkVersion Versions.min_sdk
-        targetSdkVersion Versions.target_sdk
+        minSdkVersion build_versions.min_sdk
+        targetSdkVersion build_versions.target_sdk
         resConfigs "en", "in"
-        versionCode Versions.version_code
-        versionName Versions.version_name
+        versionCode 1
```

Comment:
Why not also put both `versionCode` and `versionName` settings into `versions.gradle`?